docs/reference/commandline/secret_create.md

```diff
@@ -16,12 +16,11 @@ keywords: ["secret, create"]
 # secret create
 
 ```Markdown
-Usage:	docker secret create [OPTIONS] SECRET
+Usage:	docker secret create [OPTIONS] SECRET file|-
```

Comment:
Just thinking out loud; which naming should we keep for `file`? we're not consistent yet;

```
Usage:	docker import [OPTIONS] file|URL|- [REPOSITORY[:TAG]]

Import the contents from a tarball to create a filesystem image
```

versus

```
Usage:	docker build [OPTIONS] PATH | URL | -

Build an image from a Dockerfile
```